bevy_asset_loader/Cargo.toml

```diff
@@ -13,6 +13,7 @@ categories = ["game-development"]
 readme = "../README.md"
 
 [features]
+default = ["bevy/bevy_asset"]
```

Comment:
Shouldn't we add this feature on the Bevy dependency directly? If we put it to the default features, the crate does not work with `default-features = false`.

Reply:
You're probably right; very much a Rust and Cargo novice here, enjoying your work a lot though 😊 I will update the PR once I figure out how it should be!

Reply:
There, should be fixed now.